library/ssl_misc.h

```diff
@@ -1929,40 +1929,40 @@ int mbedtls_ssl_tls13_handshake_server_step(mbedtls_ssl_context *ssl);
 /*
  * Helper functions around key exchange modes.
  */
-static inline unsigned mbedtls_ssl_conf_tls13_check_kex_modes(mbedtls_ssl_context *ssl,
-                                                              int kex_mode_mask)
+static inline int mbedtls_ssl_conf_tls13_is_kex_mode_enabled(mbedtls_ssl_context *ssl,
+                                                             int kex_mode_mask)
 {
     return (ssl->conf->tls13_kex_modes & kex_mode_mask) != 0;
 }
 
 static inline int mbedtls_ssl_conf_tls13_psk_enabled(mbedtls_ssl_context *ssl)
```

Comment:
I like the name `mbedtls_ssl_conf_tls13_is_kex_mode_enabled()` above, so wonder if these should have `_is_` added?

e.g. `mbedtls_ssl_conf_tls13_is_psk_enabled()`

The ship has already sailed on short names, so we might as well make them consistent and explicit

Reply:
In that way, do you think we also need to add `_is_`  to `mbedtls_ssl_conf_tls13_some_ephemeral_enabled`?

Reply:
It would be consistent, yes